src/mem/metaslab.h

```diff
@@ -169,7 +169,7 @@ namespace snmalloc
     }
   };
 
-  Metaslab* Metaslab::from_link(SlabLink* lptr)
+  inline Metaslab* Metaslab::from_link(SlabLink* lptr)
```

Comment:
`SNMALLOC_FAST_PATH`?

Reply:
That may be desirable as well, the goal for this patch is to fix linker errors if two compilation units include this header file.  Any declaration in a header needs to be marked inline so that it becomes a COMDAT.  This was missed in review of #385 and broke Verona.